webxrlayers-1.bs

```diff
@@ -479,13 +492,13 @@ interface XRWebGLLayerFactory {
 
   readonly attribute double nativeProjectionScaleFactor;
 
-  Promise&lt;XRProjectionLayer&gt; requestProjectionLayer(GLenum textureTarget, 
+  Promise&lt;XRProjectionLayer&gt; requestProjectionLayer(XRTextureType textureTarget,
```

Comment:
These arguments should maybe be renamed to `textureType` now? "Target" is a very WebGL-centrix term that I only used originally to help better suggest which of the wide array of GL enums should be used here.

Reply:
I will make that change